Lib/ufo2ft/outlineCompiler.py

```diff
@@ -928,9 +928,12 @@ def setupTable_post(self):
         italicAngle = float(getAttrWithFallback(font.info, "italicAngle"))
         post.italicAngle = italicAngle
         # underline
-        underlinePosition = getAttrWithFallback(
-            font.info, "postscriptUnderlinePosition"
-        )
+        if "public.openTypePostUnderlinePosition" in font.lib:
```

Comment:
we could store this public.openTypePostUnderlinePosition lib key in https://github.com/googlefonts/ufo2ft/blob/main/Lib/ufo2ft/constants.py to avoid repeating the string literal multiple times

Reply:
Constant name is long, yes, but not confusing